src/ashley/views.py

```diff
@@ -49,6 +50,9 @@ def _do_on_login(self, lti_request: LTI) -> HttpResponse:
         # Synchronize the user groups related to the current LTI context
         context.sync_user_groups(self.request.user, lti_request.roles)
 
+        # Store the current user LTI context in session
+        self.request.session[SESSION_LTI_CONTEXT_ID] = context.id
```

Comment:
Your are now testing the middleware and the permission handler but I didn't see how you ensure that this line works? Do the tests succeed if you remove it?

Reply:
You're right. This line, or more precisely this class,  is not tested.
I'll add a test for the ForumLTIView.

Reply:
@sampaccoud I added tests for the `ForumLTIView` class.
And at the same time, I discovered/fixed a security issue. Thanks ! :pray:

Reply:
:medal_sports: